src/main/java/com/jalasoft/jfc/Main.java

```diff
@@ -11,16 +11,27 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+
```

Comment:
delete this line

Reply:
that line is correct, because the separation is for groups of imports